classes/class4/jessicas-burgers/views/home.handlebars

```diff
@@ -0,0 +1,153 @@
+<h1>
+  <center>
+
+  <img  src="../images/mainlogo.gif" >
+
+  <div class = "heading-title-banner">
+    Handcrafted
+  </div>
+  <div class = "heading-title">
+    Jessica's Burger
+  </div>
+  <div class = "heading-title-banner">
+    since 2015
+  </div>
+
+  <br><br>
+
+  <a class="arrow-wrap" href="#aboutus">
+    <img src="../images/arrow.png"><br>
+  </a>
+
+ </center>
+</h1>
+
+<section id="aboutus">
+
+  <div id="aboutus-section" class ="services bg-primary" style="background-color:#aa7243">
+    <div class="container">
+      <div class="row text-center">
+        <div class="col-lg-10 col-lg-offset-1">
+
+          <div class="row">
+            <div class="col-lg-12">
+              <h2 style="font-family: 'Myriad Set Pro', 'Lucida Grande', 'Helvetica Neue', Helvetica, Arial, Verdana, sans-serif;">"The next big thing is here." <i><h4>Gastronomy Times</h4></i></h2>
```

Comment:
Also, rather than repeating all of these, why don't you make this a template and set the reviews as a JSON array?